docs/user/exercises/programming-exercise-setup.rst

```diff
@@ -116,11 +116,48 @@ Exercise Creation
 
 3. **Update exercise code in repositories**
 
-- **Alternative 1:** Clone the 3 repositories and adapt the code on your local computer in your preferred development environment (e.g. Eclipse)
+- **Alternative 1:** Clone the 3 repositories and adapt the code on your local computer in your preferred development environment (e.g. Eclipse).
 
   - To execute tests, copy the template (or solution) code into a folder **assignment** in the test repository and execute the tests (e.g. using maven clean test)
   - Commit and push your changes |submit|
 
+  - **Notes for Haskell:** In addition to the assignment folder, the executables of the build file expect the solution repository checked out in the **solution** subdirectory of the test folder and also allow for a **template** subdirectory to easily test the template on your local machine.
+    You can use the following script to conveniently checkout an exercise and create the right folder structure:
+
+.. code-block:: bash
```

Comment:
GitHub Actions is now complaining here: `Error in "code-block" directive`
(For some reason this warning not visible as comment but just when switching to the diff view.)

Reply:
Seems to be fixed now. I missed a newline.